FIT-16

dmitryprikhunov opened the pull request:
- Added Recipe Controller, Recipe Service, Recipe Repository, Recipe Entity by Pavel Khramov
- Swagger integration for APIs (some useful annotations and comments)

dmitryprikhunov committed:
- Added Recipe Controller, Recipe Service, Recipe Repository, Recipe Entity by Pavel Khramov
- Swagger integration for APIs (some useful annotations and comments)

dmitryprikhunov committed:
- Исправлены методы на ResponseEntity
- Добавлены JavaDoc
- Переименованы классы и интерфейсы
- Убраны ненужные IF

ws-eliseev commented on src/main/resources/db/master.xml:
```diff
@@ -3,5 +3,14 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
-
+    <changeSet id="1" author="Liquibase">
```
Убрать

ws-eliseev commented on src/main/java/ws/eliseev/fitness/service/IRecipeService.java:
```diff
@@ -0,0 +1,19 @@
+import java.util.Optional;
+
+public interface IRecipeService {
+    //Вывести список всех рецептов
```
/**

ws-eliseev commented on src/main/java/ws/eliseev/fitness/model/Recipe.java:
```diff
@@ -0,0 +1,75 @@
+import javax.persistence.Id;
+import java.util.Objects;
+
+
```
javadoc

ws-eliseev commented on src/main/java/ws/eliseev/fitness/controller/RecipeController.java:
```diff
@@ -0,0 +1,107 @@
+                    description = "Got all recipes found by defined name"
+            )
+    })
+    @GetMapping("/recipes/name/{name}")
```
recipes вынести наверх

dmitryprikhunov committed:
- Добавлены недостающие Javadoc
- Переведены некоторые комментарии
- @RequestMapping в контроллере (убран повторяющийся код)
- Убран xml для бд

dmitryprikhunov committed:
- Удален Recipe

dmitryprikhunov committed:
Merge branch 'devel' into FIT-16

dmitryprikhunov committed:
- Добавлены комментарии в IRecipeService
